### DISCUSSION TITLE
English

### DISCUSSION THREAD

**Discussion by FreshImmuc** [YOURLS/YOURLS#2930] [Q&A] [answered]
Hello. How do I use english for my YOURLS? theres no english thing listed in the Translations section

**Comment by ozh** [comment, marked as answer]
Errrr... isn't the whole stuff in English out of the box ?

**Reply by FreshImmuc** [reply]
I have to choose one out of [these ](https://github.com/YOURLS/awesome-yourls#translations-) here in the config and english isnt listed

**Reply by ozh** [reply]
```
/** YOURLS language
 ** Change this setting to use a translation file for your language, instead of the default English.
 ** That translation file (a .mo file) must be installed in the user/language directory.
 ** See http://yourls.org/translations for more information */
define( 'YOURLS_LANG', '' );
```

Emphasis : **Change this setting to use a translation file for your language, _instead of the default English_.**
